XamarinCommunityToolkit/Behaviors/Validators/NumericValidationBehavior.shared.cs

```diff
@@ -22,9 +30,42 @@ public double MaximumValue
 			set => SetValue(MaximumValueProperty, value);
 		}
 
+		public int MinimumDecimalPlaces
+		{
+			get => (int)GetValue(MinimumDecimalPlacesProperty);
+			set => SetValue(MinimumDecimalPlacesProperty, value);
+		}
+
+		public int MaximumDecimalPlaces
+		{
+			get => (int)GetValue(MaximumDecimalPlacesProperty);
+			set => SetValue(MaximumDecimalPlacesProperty, value);
+		}
+
+		protected override object DecorateValue()
+			=> base.DecorateValue()?.ToString()?.Trim();
+
 		protected override bool Validate(object value)
-			=> double.TryParse(value?.ToString(), out var numeric)
+		{
+			var valueString = value as string;
+			if (!(double.TryParse(valueString, out var numeric)
 				&& numeric >= MinimumValue
-				&& numeric <= MaximumValue;
+				&& numeric <= MaximumValue))
+				return false;
+
+			var decimalDelimetrIndex = valueString.IndexOf(decimalPartDelimeter);
```

Comment:
Looks like a little typo in this variable name :D

Reply:
fixed on fly)